models/topic.go

```diff
@@ -39,6 +39,24 @@ type TopicLinks struct {
 	Content   *LinkObject `bson:"content,omitempty"    json:"content,omitempty"`
 }
 
+// PublicSubtopics ... !!! sort rest of this comment
+type PublicSubtopics struct {
+	Count       int      `bson:"count,omitempty"        json:"count"`
+	Offset      int      `bson:"offset_index,omitempty" json:"offset_index"`
+	Limit       int      `bson:"limit,omitempty"        json:"limit"`
+	TotalCount  int      `bson:"total_count,omitempty"  json:"total_count"`
+	PublicItems []*Topic `bson:"items,omitempty"        json:"items"` //!!! is items a good name ?
```

Comment:
I think the name is fine for API responses. It's actually in the standards for API list responses, to make them consistent: https://github.com/ONSdigital/dp/blob/master/standards/API_STANDARDS.md

The comment can be removed from both `PublicSubtopics` and `PrivateSubtopics`

Reply:
done